File: matiasmasso/Mat.NET/DT2/DTOBlogger.vb
Public Class DTOBlogger
    Inherits DTOBaseGuid

    Property Title As String
    Property Author As DTOUser
    Property Url As String
    <JsonIgnore> Property Logo As Image '150x115
    Property Posts As List(Of DTOBloggerPost)
    Property Obsoleto As Boolean

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

    Public Function LogoSegmentUrl() As String
        Return String.Format("Bloggers/logo", MyBase.Guid.ToString())
    End Function
End Class

File: matiasmasso/Mat.NET/DTO/DTOBlogger.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace DTO
{
    public class DTOBlogger : DTOBaseGuid
    {
        public string Title { get; set; }
        public DTOUser Author { get; set; }
        public string Url { get; set; }
        [JsonIgnore]
        public Byte[] Logo { get; set; } // 150x115
        public List<DTOBloggerPost> Posts { get; set; }
        public bool Obsoleto { get; set; }

        public DTOBlogger() : base()
        {
        }

        public DTOBlogger(Guid oGuid) : base(oGuid)
        {
        }

        public string LogoSegmentUrl()
        {
            return string.Format("Bloggers/logo", base.Guid.ToString());
        }
    }
}
